app/src/main/java/com/antonioleiva/bandhookkotlin/ui/util/ViewExtensions.kt

```diff
@@ -40,4 +43,13 @@ fun View.animateTranslationY(translationY: Int, interpolator: Interpolator) {
  */
 fun View.singleClick(l: (android.view.View?) -> Unit){
     setOnClickListener(SingleClickListener(l))
+}
+
+fun View.setTransitionGroupCompat(isTransitionGroup: Boolean) {
+    if (this is ViewGroup) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)
+            setTransitionGroup(isTransitionGroup)
+        else
+            ViewGroupCompat.setTransitionGroup(this, isTransitionGroup)
```

Comment:
Why don't you just use this line in all situations? I'd bet that the version check you're doing is exactly what `ViewGroupCompat.setTransitionGroup` is doing if you see the code.